tests/test_e2e/test_pebble.py

```diff
@@ -9,8 +9,8 @@
 from ops.pebble import ExecError, ServiceStartup, ServiceStatus
 
 from scenario import Context
-from scenario.state import Container, ExecOutput, Mount, Port, State
-from tests.helpers import jsonpatch_delta, trigger
+from scenario.state import Container, Exec, Mount, Port, State
+from tests.helpers import rigger
```

Comment:
Oops, bad merge fix.

```suggestion
from tests.helpers import trigger
```